src/main/scala/com/workday/warp/persistence/CorePersistenceAware.scala

```diff
@@ -130,57 +129,40 @@ trait CorePersistenceAware extends PersistenceAware {
       * Creates, inserts, and returns a [[TestExecutionRowLike]]
       *
       * @param testId id of the measured test (usually fully qualified junit method).
-      * @param documentation option containing documentation for the [[TestExecutionRow]]
       * @param timeStarted time the measured test was started.
       * @param responseTime observed duration of the measured test (seconds).
       * @param maxResponseTime maximum allowable response time set on the measured test (seconds).
+      * @param maybeDocs containing documentation for the [[TestExecutionRow]]
       * @return a [[TestExecutionRowLike]] with the given parameters.
       */
     override def createTestExecution(testId: String,
-                                documentation: Option[String],
-                                timeStarted: Date,
-                                responseTime: Double,
-                                maxResponseTime: Double): TestExecutionRowLike = {
+                                     timeStarted: Instant,
+                                     responseTime: Double,
+                                     maxResponseTime: Double,
+                                     maybeDocs: Option[String] = None): TablesLike.TestExecutionRowLike = {
```

Comment:
moved `maybeDocs` to the final position because it has a default value.
at invocation site, if a parameter is omitted, all following parameters must be named

https://docs.scala-lang.org/tour/default-parameter-values.html